### PR TITLE
device detection added

### PR DESCRIPTION
**Pull request by helen-dikareva**
/cc @AlexanderMoskovkin  @churkin 
it's required for https://github.com/DevExpress/testcafe/pull/1052

### CODE REVIEW

File: src/client/utils/browser.js [hunk 1]
```diff
@@ -1,50 +1,30 @@
...
 
 var userAgent = navigator.userAgent.toLowerCase();
-var browser   = calculateBrowserAndVersion(userAgent);
+var info      = bowser._detect(userAgent);
 
 //Platforms
-export var isMacPlatform = /^Mac/.test(navigator.platform);
-export var isAndroid     = /(android)/.test(userAgent);
-export var isIOS         = /(iphone|ipod|ipad)/.test(userAgent);
+export var isMacPlatform = !!info.mac;
```
Comment (helen-dikareva) [inline src/client/utils/browser.js]: because info has 'undefined' for this fields, but TestCafe expect `false`

Comment (churkin) [inline src/client/utils/browser.js]: Ok, I see now

**Comment by helen-dikareva** [comment]
publish new version please

**Pull request merged**